src/main/java/bio/terra/cli/command/helperclasses/BaseCommand.java

```diff
@@ -18,15 +21,31 @@
  *
  * <p>Sub-classes define how to execute the command (i.e. the implementation of {@link #execute}).
  */
+@CommandLine.Command
+@SuppressFBWarnings(
+    value = {"ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD", "MS_CANNOT_BE_FINAL"},
+    justification =
+        "Output streams are static, so the OUT & ERR properties should also be so. Each command "
+            + "execution should only instantiate a single BaseCommand sub-class. So in practice, "
+            + "the call() instance method will only be called once, and these static pointers to "
+            + "the output streams will only be initialized once. And since picocli handles instantiating "
+            + "the command classes, we can't set the output streams in the constructor and make them "
+            + "static.")
 public abstract class BaseCommand implements Callable<Integer> {
   protected GlobalContext globalContext;
   protected WorkspaceContext workspaceContext;
 
-  // output stream to use for writing command return value
-  protected static final PrintStream OUT = System.out;
+  // output streams for commands to write to
+  protected static PrintWriter OUT;
```

Comment:
This may be the first time I remember seeing `protected static`. You could alternatively have a protected method like `getOutWriter()` that returned the OUT property, which might be more OOPy. Then these would be private instance members and you could avoid some awkwardness. They'd just happen to point to static resources.

Reply:
I prefer having sub-commands access these as properties instead of methods. `OUT.println("...command output...")` seems more readable to me than `getOutWriter().println("..command output..")`, and more similar to `System.out.println("..command output..")`, which is what this is replacing.